vendor/vtd-api/Makefile

```diff
@@ -0,0 +1,20 @@
+# Overrides will be preserved in Makefile.package.
+override SOURCE_DIR := src
+override PACKAGE_CHANNEL := cloe/stable
+override CLEAN_SOURCE_DIR := true
+
+include ../../Makefile.package
+
+export:
+	# Only try to export the vtd sources if they are actually there.
+	@( \
+		cat conanfile.py \
+		| sed -rne 's/^.*"(.*\.tgz)"$$/\1/p' \
+		| while read file; do \
+			if [ ! -f $$file ]; then \
```

Comment:
As it turned out that using symbolic links to the source file archives leads to problems during the package build, could we make sure that the files (not just links to the files) are actually there, e.g. by using `if [ ! -f $$file ] ||  [ -L $$file ];`?

Reply:
Thing is, `[ -L $$file ]` returns true even if the symlink is broken.

Reply:
Do we really need to support symlinks? Couldn't we just use hard links in this case?

Reply:
To keep the conan cache self-contained, I changed the export of the source files such that symlinks are dereferenced (s. 152254c). With that change, my previous comment on explicitly checking the source directory for symlinks is obsolete.